cadc-util/src/main/java/ca/nrc/cadc/xml/JsonOutputter.java

```diff
@@ -141,13 +141,33 @@ public void setFormat(Format fmt)
     {
         this.fmt = fmt;
     }
-    
+
+    /**
+     * Write a Document instance to a writer.
+     *
+     * @param doc Document instance to write.
+     * @param writer Writer to write to.
+     * @throws IOException if the writer fails to write.
+     */
     public void output(final Document doc, Writer writer)
         throws IOException
+    {
+        output(doc.getRootElement(), writer);
+    }
+
+    /**
+     * Write a Document instance to a writer.
+     *
+     * @param element document to write starting from element.
+     * @param writer Writer to write to.
+     * @throws IOException if the writer fails to write.
+     */
+    public void output(final Element element, Writer writer)
```

Comment:
Why does this have to be public? Some other code can write an element without a document?